requirements.txt

```diff
@@ -0,0 +1,5 @@
+black==19.3b0
+flask==1.0.4
```

Comment:
[nit] Since 'flask' is specified in the setup.py file dependencies, it shouldn't be needed here.